### DISCUSSION TITLE
500 error in the main site

### DISCUSSION THREAD

**Discussion by anmiefan** [microweber/microweber#955] [General]
i ma getting 500 error messages some times on the main site admin is fine what web server config you guys have on the your hosted ver i tried every webserver stack i ma getting some strang lag and slowness on my main site i always change the exxution time to 3000 and some other settings in php but i am still getting some starnage thing i tried ampps server and Bitnami web server stack and xampp webserver stack still strange things same 500 error messages sometimes on the main site but admin site is fine and still a bit slowness loading i hope you guys have a webserver config for this that works 

and do any of you have a pre config web server stack and all the setting all setup and  with this built in it will be so easy to get this working  fast

**Comment by phausleitner** [comment]
I think, general hosting provider with php 8.1 support should be ok... And if not, maybe ask that provider.. Managing own server can be a long journey..

**Comment by bmooresolutions** [comment]
Here is the stackscript I used to setup my webserver on ubuntu 20.02``
[StackScript.txt](https://github.com/microweber/microweber/files/9700667/StackScript.txt)

you may be able to use all or parts of it to help you figure this out.
